config/index.js

```diff
@@ -45,8 +45,9 @@ module.exports = {
   },
   node: {
     zmq: process.env.ZMQ || 'tcp://127.0.0.1:43332',
-    network: process.env.NETWORK || 'regtest',
+    network: process.env.NETWORK || 'testnet',
     ipcName: process.env.IPC_NAME || 'bitcoin',
-    ipcPath: process.env.IPC_PATH || '/tmp/'
+    ipcPath: process.env.IPC_PATH || '/tmp/',
```

Comment:
i've asked you to replace ipcPath + ipcName to single one param, which could be ipc or of http type